<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupplierWindow</class>
 <widget class="QWidget" name="SupplierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>1081</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="i_add_supplier">
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>180</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter un fournisseur</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="supplierComboBox_2">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>290</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>230</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="supplierTable_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>561</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="supplierInput_3">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>230</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addSupplier">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>240</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="j_modif_supplier">
    <widget class="QSpinBox" name="supplierId_3">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>240</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>220</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saisissez l'id du fournisseur que vous souhaitez modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_5">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>280</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modif_3">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>410</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="supplierInput_4">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>360</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>370</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fournisseur :</string>
     </property>
    </widget>
    <widget class="Line" name="line_15">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>320</y>
       <width>641</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="msg_5">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>260</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>130</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier un fournisseur</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="supplierTable_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>551</width>
       <height>561</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="k_delete_supplier">
    <widget class="QSpinBox" name="supplierId_4">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>250</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>230</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saisissez l'id du fournisseur que vous souhaitez supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg_6">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>270</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="search_6">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>290</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteSupplier_2">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>380</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="supplierTable_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>611</width>
       <height>531</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
